packages/trackerless-network/src/logic/StreamrNode.ts

```diff
@@ -284,6 +284,20 @@ export class StreamrNode extends EventEmitter<Events> {
         return false
     }
 
+    // Returns info of all streamParts or all given streamParts that are acting as a full node on a stream
+    getInfo(): StreamPartitionInfo[] {
+        const onStreamAsFullNode = Array.from(this.streamParts.entries()).filter(([_, node]) => node.proxied === false)
```

Comment:
Rename variable? E.g. `nonProxiedStreamParts`?

Reply:
not sure if this is a good name since the node could be acting as a proxy on the stream

Reply:
I think we may need a comment here since AFAIK we don't have a good term encompassing this idea.